client/gutenberg/extensions/publicize/form-unwrapped.jsx

```diff
@@ -94,12 +59,11 @@ class PublicizeFormUnwrapped extends Component {
 		return (
 			<div id="publicize-form">
 				<ul>
-					{ staticConnections.map( c => (
+					{ connections.map( c => (
```

Comment:
If we're altering all these, let's change `c` to the more explicit `connection`.

Reply:
As discussed on Slack, I'll do that in a follow-up PR.